samples/bluetooth/mesh/light/README.rst

```diff
@@ -43,10 +43,10 @@ The configuration overlay :file:`overlay-dfu.conf` enables DFU support in the ap
 
 * nrf52833dk_nrf52833
```

Comment:
What about nrf21540dk_nrf52840?

Reply:
Since nrf21540dk_nrf52840 enables CONFIG_SOC_SERIES_NRF52X, it can most likely run the light sample with this overlay. However, Anders has not tested this.

Btw, does anyone on the team have that board?

Reply:
I have updated the nrf21540dk_nrf52840 board. So I think we should add it.